src/client/modules/my/dataUpload/dataUpload.ts

```diff
@@ -19,7 +32,17 @@ export default class DataUploader extends LightningElement {
         //gets value from textarea
         let textBlob: string = this.getValueFromInput('textArea');
         let dataList: GameData[] = this.processLine(textBlob);
+        let gameIds: string[] = [];
+        for (let game of dataList) {
+            gameIds.push(game.gameId);
+        }
         let errorMessages = validateInput(dataList);
+        //get file values
+        let fileString = '';
+        let fileText = this.template.querySelector(
+            'input[name="file-upload-input-107"]'
```

Comment:
Name is not static AFAIK, so we may want to explicitly set it, or look at a different prop. I'll open an issue and fix after